chromatic/rainbows/actions/remove_astrophysical_signals.py

```diff
@@ -0,0 +1,75 @@
+from ...imports import *
+
+
+def remove_astrophysical_signals(self, method="gradient", model=None, win_length=None, polyorder=None, butter_cutoff=None, butter_fs=None, butter_order=9):
```

Comment:
I really like this function, @Pat-Wachiraphan ! I'm going to make some small changes to this so that it will define a reasonable default for each method.